static/src/javascripts/projects/common/modules/email/run-checks.js

```diff
@@ -97,7 +98,8 @@ define([
             return page.keywordExists(['Football']) && allowedArticleStructure();
         },
         usBriefing: function () {
-            return config.page.section === 'us-news' && allowedArticleStructure();
+            return (config.page.section === 'us-news' || config.page.series === 'Guardian US briefing') &&
```

Comment:
Yeah, it's from the series tag in composer: 
![image](https://cloud.githubusercontent.com/assets/638051/15293077/24b91f4a-1b7e-11e6-8708-299fad980e8a.png)